src/storage/perf_tracker.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use storage::engine::{PerfStatisticsDelta, PerfStatisticsInstant};
```

Comment:
@breeswish I have a question for PerfStatisticsInstant, if there are multiple parallel get/scan requests in RocksDB, PerfStatisticsInstant only statistic for one operation or all the running requests?

Reply:
@zhangjinpeng1987 It's thread local